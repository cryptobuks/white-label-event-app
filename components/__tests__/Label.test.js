import React from 'react';
import renderer from 'react-test-renderer';
import { Label } from '../';

describe('Label', () => {
  it('renders correctly', () => {
    const comp = <Label value="labelValue" />;

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
