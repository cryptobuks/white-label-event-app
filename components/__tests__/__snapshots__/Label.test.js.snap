// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScheduleItemHeader renders correctly 1`] = `
<View
  style={
    Object {
      "height": 40,
      "justifyContent": "space-between",
      "width": 200,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontSize": 16,
        "fontWeight": "600",
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontSize": 14,
        "fontWeight": "600",
      }
    }
  />
</View>
`;
